app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -44,24 +46,18 @@ class SearchActivity : AppCompatActivity() {
         // подгружаем список "Вы искали"
         historyTracks = searchHistory.readHistory()
 
-        // завешиваем адаптер для списка "Вы искали" с пустым клик листнером т.к. действия сним пока не нужны
+        // завешиваем адаптер для списка "Вы искали" с листнером (пока таким же как основной)
         val onHistoryTrackClickListener = object : TracksAdapter.OnItemClickListener {
-            override fun onItemClick(track: Track) {}
+            override fun onItemClick(track: Track) {
+                searchHistory.addTrackToHistoy(track)
```

Comment:
🍏 Закралась опечатка:
```suggestion
                searchHistory.addTrackToHistory(track)
```

Reply:
пофиксил, спасибо)